python/singa/image_tool.py

```diff
@@ -115,6 +116,11 @@ def crop_and_resize(img, patch, position):
     # print "crop to box %d,%d,%d,%d and scale to %d,%d" % (box+crop)
     return new_img
 
+def resize_tuple(img,patch):
```

Comment:
this function is too simple to exist.
you can call `img.resize(path)` directly.
instead of calling `resize_tuple`.